pronotepy/dataClasses.py

```diff
@@ -619,6 +644,95 @@ def files(self):
         return [File(self._client, jsn) for jsn in self._files]
 
 
+class Information:
+    """
+    Represents a information in a information and surveys tab.
+
+    Attributes
+    ----------
+    id : str
+        the id of the information
+    author : str
+        author of the information
+    title : str
+        title of the information
+    content: str
+        content of the information
+    read : bool
+        if the message has been read
+    creation_date : datetime.datetime
+        the date when the message was created
+    start_date : datetime.datetime
+        the date when the message became visible
+    end_date : datetime.datetime
+        the date on which the message will be withdrawn
+    category: str
+        category of the information
+    survey: bool
+        if the message is a survey
+    anonymous_response : bool
+        if the survey response is anonymous
+    """
+
+    id: str
+    title: str
+    author: str
+    content: str
+    read: bool
+    creation_date: datetime.datetime
+    start_date: datetime.datetime
+    end_date: datetime.datetime
+    category: str
+    survey: bool
+    anonymous_response: bool
+
+    __slots__ = ['id', 'title', 'author', 'read', 'creation_date', 'start_date', 'end_date', 'category',
+                 'survey', 'anonymous_response', '_raw_content', '_client']
+
+    attribute_guide = {
+        'N': ('id', str),
+        'L': ('title', str),
+        'auteur': ('author', str),
+        'listeQuestions,V': ('_raw_content', list),
+        'lue': ('read', bool),
+        'dateCreation,V': ('creation_date', Util.date_parse),
+        'dateDebut,V': ('start_date', Util.date_parse),
+        'dateFin,V': ('end_date', Util.date_parse),
+        'categorie,V,L': ('category', str),
+        'estSondage': ('survey', bool),
+        'reponseAnonyme': ('anonymous_response', bool)
+    }
+
+    def __init__(self, client, json_):
+        self._client = client
+        prepared_json = Util.prepare_json(self.__class__, json_)
+        for key in prepared_json:
+            self.__setattr__(key, prepared_json[key])
+
+    @property
+    def content(self):
+        return Util.htlm_parse(self._raw_content[0]['texte']['V'])
+
+    def mark_as_read(self, status: bool):
+        data = {
+            "listeActualites": [
+                {
+                    "N": self.id,
+                    "validationDirecte": True,
+                    "genrePublic": 4,
+                    "public": {
+                        "N": self._client.info.id,
+                        "G": 4,
+                    },
+                    "lue": status,
+                }
+            ],
+            "saisieActualite": False
+        }
+        if self._client.post("SaisieActualites", 8, data):
```

Comment:
Does this work? On the demo website the returned dict is empty, thus this would always be `False`, but also the change isn't recorded, so it actually might be the right behaviour. If it isn't, please change it in `Homework.set_done` too.

Reply:
the post request always returns this:
`{'nom': 'SaisieActualites', 'session': 8672986, 'numeroOrdre': 'CE1865AEEFEC6994477CEF8FD178FF42', 'donneesSec': {'donnees': {}, 'nom': 'SaisieActualites', 'RapportSaisie': {}}}`

which is equivalent to True

Reply:
Oh, right! Well then we basically can get rid of the check, no? The `post` method would throw an exception if it failed.

Reply:
absolutely